utils/process-stats-dir.py

```diff
@@ -415,6 +258,37 @@ def compare_to_csv_baseline(args):
     return regressions
 
 
+def compare_to_csv_baseline(args):
+    old_stats = read_stats_dict_from_csv(args.compare_to_csv_baseline)
+    sel = args.select_module
+    m = merge_all_jobstats([s for d in args.remainder
```

Comment:
Can't you drop the [] here?

Reply:
yeah, huon keeps trying to teach me to do that :)

Reply:
Hah, actually ...

    SyntaxError: Generator expression must be parenthesized if not sole argument

Learn something every day!